python/foglamp/services/common/microservice.py

```diff
@@ -67,7 +68,7 @@ def _make_microservice_management_app(self):
         # create web server application
         self._microservice_management_app = web.Application(middlewares=[middleware.error_middleware])
         # register supported urls
-        routes.setup(self._microservice_management_app)
+        routes.setup(self._microservice_management_app, self)
```

Comment:
A common routes.py services both core and non-core microservice's management service needs. All that is passed in is the object (or class) that holds the callbacks (not to be confused with configuration callbacks) that need to be called.